spec/models/post_spec.rb

```diff
@@ -1,5 +1,51 @@
 require 'rails_helper'
 
 RSpec.describe Post, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'validates' do
+    subject { Post.new title: 'RSpec', user_id: 0 }
+
+    before { subject.save }
+
+    it 'should have a title' do
+      subject.title = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'should have title with at most 250 characters' do
+      subject.title = 'RoR' * 250
+      expect(subject).to_not be_valid
+    end
+
+    it 'should not have a blank title' do
+      subject.title = ' '
+      expect(subject).to_not be_valid
+    end
+
+    it 'should have a postive integer comments counter' do
+      subject.comments_counter = -1
+      expect(subject).to_not be_valid
+    end
+
+    it 'should have a postive integer likes counter' do
+      subject.likes_counter = -1
+      expect(subject).to_not be_valid
```

Comment:
- [ ] Remember to change this test as well to accept integers which are greater than zero

Reply:
Why? @DeliceLydia Isn't the point of test to reject invalid values?